src/ert/_c_wrappers/enkf/ensemble_config.py

```diff
@@ -389,7 +389,7 @@ def get_surface_node(surface: List[str]) -> SurfaceConfig:
         elif not Path(base_surface).exists():
             errors.append(f"BASE_SURFACE:{base_surface} not found")
         if errors:
-            errors = "\n".join(errors)
+            errors = ". ".join(errors)
```

Comment:
Not sure how this will show in the suggestor dialog but would it make sense to separate the errors with `;`? It might be a bit easier to separate them when reading multiple errors.

Reply:
Makes sense, and done :)